__tests__/commands/install/bin-links.js

```diff
@@ -20,6 +21,36 @@ async function linkAt(config, ...relativePath): Promise<string> {
   }
 }
 
+function execCommand(cwd: string, binPath: Array<string>, args: Array<string>): Promise<Array<?string>> {
```

Comment:
I remember seeing a function like this in somewhere else. May be we should start using something like https://yarnpkg.com/en/package/jsinspect to help catch these automatically? (also it would help me be sure instead of saying "I suspect" :D)